userstories.md

```diff
@@ -0,0 +1,50 @@
+- Who specifically will use a Portal client?
+- What are they using Portal to accomplish, or what pain are they trying to avoid?
+- What is their current best alternative, and where does it fall short?
+
+
```

Comment:
Just throwing this in here as a potential direction for an additional user: archive node user who doesn't have space for a full archive:

- #193